.github/workflows/tests.yml

```diff
@@ -8,15 +8,14 @@ on:
 jobs:
   build_and_test:
     runs-on: ubuntu-latest
-
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
       - name: Install Node
         uses: actions/setup-node@v2
         with:
-          node-version: '10.16.3'
+          node-version: 12
```

Comment:
Is there anything that stops us from going with v14? It looks like v12 will be maintained for only 3 more months, according to [this](https://endoflife.date/nodejs).

Reply:
The short answer is "I don't know" :laughing: so I'd rather have a gentler bump to 12 and see if anything explodes and then revisit to 14 later.